Makefile

```diff
@@ -92,6 +92,10 @@ else
 	CFLAGS += -DFCH_YANGTZEE
 endif
 
+ifeq ($(IPXE),y)
```

Comment:
Not needed. Please take a look at: https://github.com/pcengines/sortbootorder/pull/81/files#diff-4062b3f978b19a82ecef89231f80c0e646b86174e6587d0c82ff03277838416dR156
Here the mainboard name is retrieved. Just strcmp it to apu7 and make a variable called `no_pxe` or something like that and simply do not handle any options regarding PXE.

Reply:
Done as per suggestion.